packages/terra-form-select/tests/jest/__snapshots__/Frame.test.jsx.snap

```diff
@@ -27,11 +27,23 @@ exports[`Frame should render a clear option 1`] = `
       "messages": Object {
         "Terra.form.select.add": "Add \\"{text}\\"",
         "Terra.form.select.ariaLabel": "Search",
+        "Terra.form.select.clearSelect": "Clear select",
         "Terra.form.select.defaultDisplay": "- Select -",
+        "Terra.form.select.defaultUsageGuidance": "Use up and down arrow keys to navigate through options. Press enter to select an option.",
+        "Terra.form.select.dimmed": "Dimmed.",
+        "Terra.form.select.disabled": "Disabled.",
+        "Terra.form.select.listOfTotalOptions": "List of {total} options.",
         "Terra.form.select.maxSelectionHelp": "Limit {text} items.",
         "Terra.form.select.maxSelectionOption": "Maximum number of {text} items selected",
+        "Terra.form.select.mobileButtonUsageGuidance": "Tap to navigate to options.",
+        "Terra.form.select.mobileUsageGuidance": "Swipe right to navigate to options.",
+        "Terra.form.select.multiSelectUsageGuidance": "Enter text or use up and down arrow keys to navigate through options. Press enter to select or unselect an option.",
         "Terra.form.select.noResults": "No matching results for \\"{text}\\"",
-        "Terra.form.select.selected": "{text} Selected",
```

Comment:
How does changing the intention of a translation key's meaning affect downstream consumers if they are providing custom translations?

I see here we updated the `Terra.form.select.selected` key meaning to be `Terra.form.select.selectedText` and then changed `Terra.form.select.selected`to have a different representation & string translations.

I don't think updating translations are a concern, but wasn't sure of the key intent.

Reply:
`Terra.form.select.selected` is a new key added in this PR. I have another PR which adds only the new translation keys so people can see what new translations are being added. https://github.com/cerner/terra-core/pull/2266/files